test/ng/documentSpec.js

```diff
@@ -27,3 +27,29 @@ describe('$document', function() {
     });
   });
 });
+
+
+describe('$$isDocumentHidden', function() {
+  it('should listen on the visibilitychange event', function() {
+    var doc;
+
+    var spy = spyOn(document, 'addEventListener').and.callThrough();
+
+    inject(function($$isDocumentHidden, $document) {
+      expect(spy.calls.mostRecent().args[0]).toBe('visibilitychange');
+      expect(spy.calls.mostRecent().args[1]).toEqual(jasmine.any(Function));
+      expect($$isDocumentHidden()).toBeFalsy(); // undefined in browsers that don't support visibility
```

Comment:
This seems slightly unrelated to the rest of the testcase. Could as well go into it's own testcase.